teleop/src/ext/InterruptableSleep.h

```diff
@@ -0,0 +1,63 @@
+#pragma once
+
+#include <chrono>
+#include <thread>
+#include <future>
+#include <mutex>
+#include <sstream>
+
+namespace Bosma {
```

Comment:
Does this mean that Bosma is still a part of the examples? (this will cause some licensing issues. as far as I remember the Bosma stuff is a third party library and we have to handle it with care, like we did with rxcpp).

Reply:
Yes, it's still part of these packages, as it's used in one (or more) node(s).

Reply:
As to the licensing: it's all MIT and Apache 2, so we'd need to state that in the manifests and/or readme, but otherwise we should be OK.